modules/installing-gcp-user-defined-labels-and-tags.adoc

```diff
@@ -0,0 +1,123 @@
+// Module included in the following assemblies:
+// * installing/installing_gcp/installing-gcp-customizations.adoc
+
+:_content-type: CONCEPT
+[id="installing-gcp-user-defined-labels-and-tags_{context}"]
+= Configuring the user-defined labels and tags for GCP
+
+
+GCP provides tags that help to identify and organize the resources created for a specific {product-title} cluster, making them easier to manage. 
+
+You can define labels & tags on the GCP resources only during cluster creation that is available from {product-title} version {product-version}. User-defined labels and tags are not supported for {product-title} clusters upgraded to {product-title} version {product-version}.
+
+User-defined labels and {product-title} specific labels are applied only to resources created by {product-title} installer and its core operators such as `machine-api-provider-gcp`, `cluster-image-registry-operator`, `gcp-filestore-csi-driver-operator`, `gcp-pd-csi-driver-operator` and not on the resources created by any other operators or the kubernetes in-tree components that creates resources for example the ingress load balancers.
```

Comment:
{product-title}/ installation program

![Screenshot from 2023-08-28 16-05-14](https://github.com/openshift/openshift-docs/assets/57954076/ec6e836d-2571-4f0d-993a-61c79caf0437)

I think the core operators belong in a list. Large paragraph should be broken into at least two sentences. 

ingress load balancer/Ingress loab balancer. 

![Screenshot from 2023-08-28 16-06-35](https://github.com/openshift/openshift-docs/assets/57954076/412b33e5-a4cf-4434-a55a-9b75208deb3e)

Reply:
s/in-tree components that creates/in-tree components that create

Reply:
Ack.